docs/eventing/event-registry.md

```diff
@@ -14,7 +14,7 @@ type information in the cluster data store.
 
 ## Before you begin
 
-1. Read about the [broker](./broker/) and [trigger](./triggers/) objects.
+1. Read about the [broker](./broker/) and [trigger](./broker/triggers/) objects.
```

Comment:
```suggestion
1. Read about the [broker](./broker) and [trigger](./broker/triggers) objects.
```
I think is matter of removing the `/` but we can do a pass and run a tool to detect all broken links later